bin/cake.php

```diff
@@ -16,17 +16,8 @@
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
 
-$minVersion = '5.6.0';
-if (file_exists('composer.json')) {
```

Comment:
I had this thought too, and i think even in CLI this is relevant if you write a cli based app with tons of calls. So it is aside the elegance of reading from composer ... better anyway.